apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/Vedtak.kt

```diff
@@ -77,15 +77,24 @@ data class Vedtak(
             )
         }, // sammendraget bør lages av beregning
         pensjonTilUtbetaling = utbetalingsperioder,
-        vedtakFattet = this.saksbehandlerId?.let { ansvarligSaksbehandler ->
-            VedtakFattet(
-                ansvarligSaksbehandler,
-                "0000",
-                this.datoFattet?.atZone(
-                    ZoneOffset.UTC
-                )!!
+        vedtakFattet = if (this.vedtakStatus in listOf(
+                VedtakStatus.FATTET_VEDTAK,
+                VedtakStatus.ATTESTERT,
+                VedtakStatus.IVERKSATT
             )
-        }, // logikk inn der fatting skjer. DB utvides med enhet og timestamp?
+        ) {
+            this.saksbehandlerId?.let { ansvarligSaksbehandler ->
+                VedtakFattet(
+                    ansvarligSaksbehandler,
+                    "0000",
```

Comment:
Dette vil kræsje brev siden brev er avhengig av å få en gyldig enhet. Hvis vi skal tillate `0000` burde vi i så fall diskutere hvordan det eventuelt skal håndteres.

Reply:
Denne har vel vært her hele tiden, så foreslår å lage en task på det (og ikke en blocker for denne oppgaven)